stormpath/express-stormpath
# Add web.spa option support

Adds support for the web.spa option.

#### Before reviewing

This depends on #313 to be merged before being reviewed.

#### How to verify

1. Checkout this branch.
2. Run `$ npm link`.
3. Checkout the [React example app](https://github.com/stormpath/stormpath-express-react-example).
4. Run `$ npm link express stormpath`.
5. Open up `server.js`.
6. Remove the old `web.spaRoot` option.
7. Set `web.spa.enabled` to `true` and `web.spa.view` to `path.join(__dirname, 'build/index.html')`.
8. Run `$ npm start`.
9. Navigate to [http://locahost:3000/login](http://locahost:3000/login).
10. Verify that the React login view (`build/index.html`) is shown.
11. Open up `server.js`and set `web.spa.enabled` to `false`.
12. Run `$ npm start`.
13. Navigate to [http://locahost:3000/login](http://locahost:3000/login).
14. Verify that the Express Stormpath login view is shown.

Fixes #319.
